lib/pulse_message.js

```diff
@@ -12,7 +12,7 @@ const Model = require('./model')
 const fields = {
   id: 0,
   mts: 1,
-  // placeholder
+  parent: 2,
```

Comment:
have you tested this?

doesn't seem that the history endpoint for pulse messages returns just the overall amount of comments

Reply:
maybe i am missing something but afaik comments with parent id are not supplied via the history endpoint which uses this model?